File: FrasesdoDia/app/src/main/java/com/example/frasesdodia/MainActivity.java
package com.example.frasesdodia;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.TextView;

import java.util.Random;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    public void gerarNovaFrase(View view){

        String[] frases = {"Frase 0",
                           "Frase 1",
                           "Frase 2",
                           "Frase 3",
                           "Frase 4",
                           "Frase 5"}
        ;

        int indice = new Random().nextInt(frases.length);//(bound: 5)
        TextView texto = findViewById(R.id.textResultado);
        texto.setText( frases[indice] );


    }
}

File: AppSorteioKotlin/app/src/main/java/com/emerson/cursoandroid/appsorteiokotlin/MainActivity.kt
package com.emerson.cursoandroid.appsorteiokotlin

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.TextView
import java.util.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }

    fun sortearNumero(view: View){

        var texto = findViewById<TextView>(R.id.textoSorteio)
        var numeroSorteado = Random().nextInt(11)
        texto.text = "Número sorteado é: " + numeroSorteado

    }

}